crates/router/src/connector/bamboraapac.rs

```diff
@@ -578,3 +609,7 @@ impl api::IncomingWebhook for Bamboraapac {
         Err(report!(errors::ConnectorError::WebhooksNotImplemented))
     }
 }
+
+fn html_to_xml_string_conversion(res: String) -> String {
+    res.replace("&lt;", "<").replace("&gt;", ">")
```

Comment:
are we certain this is the only replacement required to parse?

Reply:
yup, in response, these are the only charactors that comes in html character entities